pkgs/misc/drivers/sc-controller/default.nix

```diff
@@ -24,12 +24,15 @@ buildPythonApplication rec {
 
   checkInputs = [ pytest ];
 
+  patches = [ ./fix-udev.patch ];
```

Comment:
Please add a comment that references the upstream issue/patch and reminds to remove this on the next update.

Reply:
Done.